match/hammond_pd.py

```diff
@@ -0,0 +1,42 @@
+from match.lafayette_so import prepare_post
+import sys 
+sys.path.append('../')
+import pandas as pd
+from lib.path import data_file_path, ensure_data_dir
+from datamatch import JaroWinklerSimilarity, ThresholdMatcher, ColumnsIndex
+
+
+def prepare_post_data():
+    post = pd.read_csv(data_file_path('clean/pprr_post_2020_11_06.csv'))
+    return post[post.agency == 'hammond pd']
+
+
+def match_cprr_and_post(cprr, post):
+    dfa = cprr[['uid', 'first_name', 'last_name']]git
+    dfa = dfa.drop_duplicates(subset=['uid']).set_index('uid')
+    dfa.loc[:, 'fc'] = dfa.first_name.fillna('').map(lambda x: x[:1])
+
+    dfb = post[['uid', 'first_name', 'last_name']]
+    dfb = dfb.drop_duplicates(subset=['uid']).set_index(['uid'])
+    dfb.loc[:, 'fc'] = dfb.first_name.fillna('').map(lambda x: x[:1])
+
+    matcher = ThresholdMatcher(ColumnsIndex('fc'), {
+        'first_name':  JaroWinklerSimilarity(),
+        'last_name': JaroWinklerSimilarity(),
+    }, dfa, dfb)
+    decision = .5

```

Comment:
This decision is very low, the lowest I have ever seen. I think if a pair score `.5` then the chance that the 2 records refer to the same entity is less than 50%. Be careful with this.

Reply:
Typo. Corrected.